docs/source/markdown/podmansh.1.md

```diff
@@ -52,13 +56,19 @@ RequiredBy=default.target
 _EOF
 ```
 
-Users inside of this Quadlet are allowed to become root within the user namespace, and able to read/write content in their homedirectory which is mounted from a subdir `data` of the hosts users account.
+Alternatively, while running as root, create a Quadlet where the user is allowed to become root within the user namespace. They can also permanently read/write content from their homedirectory which is volume mounted from the actuall host's users account, rather then being inside of the container.
```

Comment:
rather **than**, not then. And "home directory" is two words. I'm sorry! But I'll approve anyway, let you decide whether to fix now or in a followup.